include_core/omrport.h

```diff
@@ -228,6 +228,7 @@
 #define OMRPORT_VMEM_MEMORY_MODE_SHARE_FILE_OPEN 0x000000200
 #define OMRPORT_VMEM_MEMORY_MODE_MMAP_HUGE_PAGES 0x000000400
 #define OMRPORT_VMEM_MEMORY_MODE_DOUBLE_MAP_AVAILABLE 0x000000800
+#define OMRPORT_VMEM_MEMORY_MODE_SHARE_TMP_FILE_OPEN 0x000001000
```

Comment:
I think this deserves a comment noting that `OMRPORT_VMEM_MEMORY_MODE_SHARE_TMP_FILE_OPEN` is ignored
 when combined with `OMRPORT_VMEM_MEMORY_MODE_SHARE_FILE_OPEN`.